### PR TITLE
[MU4] Palettes stabilization

### PR DESCRIPTION
**Pull request by RomanPudashkin**

### CODE REVIEW

File: src/notation/internal/notationactioncontroller.cpp [hunk 1]
```diff
@@ -454,37 +475,57 @@ void NotationActionController::putTuplet(int tupletCount)
     if (noteInput->isNoteInputMode()) {
         noteInput->addTuplet(options);
     } else {
-        interaction->addTupletToSelectedChords(options);
+        interaction->addTupletToSelectedChordRests(options);
     }
 }
 
+void NotationActionController::addBeatToSelectedChordRests(BeamMode mode)
```
Comment (cbjeukendrup) [inline src/notation/internal/notationactioncontroller.cpp:482]: Typo: beat vs beam

**Comment by vshalkevich** [comment]
Checked Palettes. Looked good. I would like to see this Palette fixes in the Master

File: src/notation/internal/notationactions.cpp [hunk 2]
```diff
@@ -377,6 +377,38 @@ const ActionList NotationActions::m_actions = {
                QT_TRANSLATE_NOOP("action", "Append Text Frame"),
                QT_TRANSLATE_NOOP("action", "Append text frame")
                ),
+    ActionItem("acciaccatura",
+               ShortcutContext::NotationActive,
```
Comment (Eism) [inline src/notation/internal/notationactions.cpp:389]: ShortcutContext::NotationHasSelection

File: src/notation/internal/notationactions.cpp [hunk 3]
```diff
@@ -133,11 +133,10 @@ const ActionList NotationActions::m_actions = {
                QT_TRANSLATE_NOOP("action", "Select All"),
                QT_TRANSLATE_NOOP("action", "Select all")
                ),
-    ActionItem("delete",
-               ShortcutContext::NotationHasSelection,
-               QT_TRANSLATE_NOOP("action", "Delete"),
-               QT_TRANSLATE_NOOP("action", "Delete the selected element(s)"),
-               IconCode::Code::DELETE_TANK
+    ActionItem("select-section",
+               ShortcutContext::NotationActive,
```
Comment (Eism) [inline src/notation/internal/notationactions.cpp:137]: ShortcutContext::NotationHasSelection

File: src/notation/internal/notationactions.cpp (hunk 2 above)
Comment (RomanPudashkin) [reply src/notation/internal/notationactions.cpp]: Fixed in all places, thanks

**Review by Eism — APPROVED** [review]

**Pull request merged**